apps/cookbook/src/app/showcase/menu-showcase/menu-showcase.component.html

```diff
@@ -275,6 +275,60 @@ <h3>
   </div>
 </div>
 
+<h2>Accessibility</h2>
+<p>
+  The Menu implements the
+  <a
+    class="kirby-external-icon"
+    target="_blank"
+    href="https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/"
+  >
+    Menu Button Pattern from ARIA Authoring Practices Guide
+  </a>
+</p>
+<h3>Keyboard support</h3>
+<p>
+  The component has full keyboard support for opening the menu and navigating between and selecting
+  items in the menu.
+</p>
+<p>The following keys can be used to open the menu when the menu button has focus:</p>
+<!-- prettier-ignore -->
+<ul>
+  <li>
+    <kbd>Enter</kbd>, <kbd>Space</kbd> and <kbd>&#8595;</kbd> opens menu and moves focus to first menu item.
+  </li>
+  <li>
+    <kbd>&#8593;</kbd> opens menu and moves focus to last menu item.
+  </li>
+</ul>
+<p>The following keys can be used to navigate and select within the menu:</p>
+<!-- prettier-ignore -->
+<ul>
+  <li>
+    <kbd>Enter</kbd> and <kbd>Space</kbd> activates the menu item and closes the menu.
+  </li>
+  <li>
+    <kbd>Escape</kbd> closes the menu.
+  </li>
+  <li>
+    <kbd>&#8593;</kbd> moves focus to the previous menu item and if focus is on the first menu item, moves focus to the last menu item.
```

Comment:
Consider readability:
```suggestion
    <kbd>&#8593;</kbd> moves focus to the previous menu item. If focus is on the first menu item, moves focus to the last menu item.
```